stl/src/alias.asm

```diff
@@ -0,0 +1,24 @@
+; weak symbol aliases
+
+; Copyright (c) Microsoft Corporation.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; Note that while this is a masm file, it contains no code
+; masm does generate a .text section with zero size
+
+IFDEF RAX ; note: this only works with ifdef, not ifndef

```

Comment:
Should we have another `; no, seriously` comment here? (No change requested, just :boggle:)

Reply:
Serious comment: Future us would appreciate a reminder of what this syntax means. Yes, I'm always raging that comments shouldn't tell us what the code is doing, but this is _really_ obscure.

Reply:
Yeah, I actually like comments that tell us what the code is doing if the reader might lack knowledge of the language or syntax. Given MASM is pretty sparsely documented I think that is almost everyone